src/components/functional_module/include/functional_module/generic_module.h

```diff
@@ -42,6 +42,16 @@
 
 namespace functional_modules {
 
+/**
+ * @brief The SDLEvent enum defines SDL event that module can react on
+ */
+enum SDLEvent {
```

Comment:
@LuxoftAKutsan 
Didn't catch that. What is the connection between enum naming and plugin parameters? Can you show an example?